### PR TITLE
Add RGDB launch file for the ZR300

### PR DESCRIPTION
**Pull request by dbolkensteyn**
Fixes Issue: #227 

Duplicates the R200 RGBD launch file but also exposes (and enables by default) the IMU and fisheye camera of the ZR300.

### CODE REVIEW

File: realsense_camera/launch/zr300_nodelet_rgdb.launch [hunk 1]
```diff
@@ -0,0 +1,118 @@
+<!-- Sample launch file for using RealSense R200 camera with rgbd_launch -->
+<launch>
+
+  <!-- "camera" should be a user friendly string to uniquely identify the device namespace.
+       All topics are pushed down into the "camera" namespace, and it is prepended to tf frame ids. -->
+  <arg name="camera"          default="camera" />
+
+  <!-- Type of RealSense camera -->
+  <arg name="camera_type"     default="ZR300" />
```
Comment (dbolkensteyn) [inline realsense_camera/launch/zr300_nodelet_rgdb.launch:9]: this differs from the R200 RGDB launch file

File: realsense_camera/launch/zr300_nodelet_rgdb.launch [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+  <!--  ZR300 specific features -->
```
Comment (dbolkensteyn) [inline realsense_camera/launch/zr300_nodelet_rgdb.launch]: this is new compared the the R200 RGDB launch file

File: realsense_camera/launch/zr300_nodelet_rgdb.launch [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+             file="$(find realsense_camera)/launch/includes/nodelet_rgbd.launch.xml">
+      <arg name="manager"                   value="$(arg manager)" />
+      <arg name="camera"                    value="$(arg camera)" />
+      <arg name="camera_type"               value="$(arg camera_type)" />
+      <arg name="serial_no"                 value="$(arg serial_no)" />
+      <arg name="usb_port_id"               value="$(arg usb_port_id)" />
+      <arg name="depth"                     value="$(arg depth)" />
+      <arg name="rgb"                       value="$(arg rgb)" />
+      <arg name="ir"                        value="$(arg ir)" />
+      <arg name="enable_fisheye"            value="$(arg enable_fisheye)" />
```
Comment (dbolkensteyn) [inline realsense_camera/launch/zr300_nodelet_rgdb.launch]: these 2 lines are new compared the the R200 RGDB launch file

File: realsense_camera/launch/zr300_nodelet_rgdb.launch [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+      <arg name="enable_imu"                value="$(arg enable_imu)" />
```
Comment (dbolkensteyn) [inline realsense_camera/launch/zr300_nodelet_rgdb.launch:86]: this line and the one above are new

File: realsense_camera/launch/zr300_nodelet_rgdb.launch [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  <!-- "camera" should be a user friendly string to uniquely identify the device namespace.
+       All topics are pushed down into the "camera" namespace, and it is prepended to tf frame ids. -->
+  <arg name="camera"          default="camera" />
+
+  <!-- Type of RealSense camera -->
+  <arg name="camera_type"     default="ZR300" />
+
+  <!--  ZR300 specific features -->
+  <arg name="enable_fisheye"      default="true" />
+  <arg name="enable_imu"          default="true" />
```
Comment (dbolkensteyn) [inline realsense_camera/launch/zr300_nodelet_rgdb.launch:13]: this line and two ones above are new

**Review by mattcurfman — APPROVED** [review]
@dbolkensteyn, please read "Contributing to the Project" in the top level README.md, and accept the terms for contributing to this project, then I will merge this PR

**Comment by dbolkensteyn** [comment]
Thank you @mattcurfman, I do accept the terms for contributing to this project

**Pull request merged**